js/main.js

```diff
@@ -0,0 +1,39 @@
+const div = document.createElement('nav');
+const grayBgDiv = document.createElement('div');
+const hamburgerBttn = document.querySelector('.menu-bars');
+
+hamburgerBttn.addEventListener('click', () => {
+  div.id = 'overlay';
+  div.classList.add('overlay');
+  grayBgDiv.id = 'overlay-menu-bg';
+  grayBgDiv.classList.add('overlay');
+  div.innerHTML = '<img src="./media/cancel.png" class="cancelBttn"></img>'
+    + '<ul class="mobile-menu-ul">'
+    + '<li class="mobile-menu-li"><a href="#projects">Porfolio</a></li>'
+    + '<img class="overlay-bg-separator" src="./media/separator-mobile.png"></img>'
+    + '<li class="mobile-menu-li"><a href="#about-me">About</a></li>'
+    + '<img class="overlay-bg-separator" src="./media/separator-mobile.png"></img>'
+    + '<li class="mobile-menu-li"><a href="#contact">Contact</a></li>'
+    + '<img class="overlay-bg-separator" src="./media/separator-mobile.png"></img>'
+    + '</ul>';
```

Comment:
- [ ]   The hamburger menu is still not following the design template as pointed out by the previous reviewer. You need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to make the hamburger menu closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .

You can see the screenshot images for comparison. ⬇️ 

### Current mobile menu
![1](https://user-images.githubusercontent.com/80895497/147281804-276aea71-70c9-4ca3-9fe2-c9ad648dcea5.png)

 ### Original design
![original_design](https://user-images.githubusercontent.com/80895497/147281867-b9fd3f2f-881f-4451-8092-a26c6c53f37e.png)